packages/medusa/src/types/order-edit.ts

```diff
@@ -0,0 +1,27 @@
+import { OrderEdit } from "../models"
+
+export const defaultOrderEditRelations: string[] = [
+  "order",
+  "order.items",
```

Comment:
thought: if you think it is worth implementing my comment above we could remove this